$NetBSD: patch-pr3287_Makefile.obj.in,v 1.1 2023/11/21 19:19:01 rhialto Exp $

$(INSTALL_PROGRAM) tries to strip a shell script, which fails, so use
the proper $(INSTALL_SCRIPT) instead.

--- pr3287/Makefile.obj.in.orig	2023-10-14 16:30:03.000000000 +0000
+++ pr3287/Makefile.obj.in
@@ -46,6 +46,7 @@ LIBS = @LIBS@
 INSTALL = @INSTALL@
 INSTALL_PROGRAM = @INSTALL_PROGRAM@
 INSTALL_DATA = @INSTALL_DATA@
+INSTALL_SCRIPT = @INSTALL_SCRIPT@
 
 HOST = @host@
 include pr3287_files.mk libs.mk
@@ -70,7 +71,7 @@ install: pr3287 prtodir
 	[ -d $(DESTDIR)$(BINDIR) ] || \
 		mkdir -p $(DESTDIR)$(BINDIR)
 	$(INSTALL_PROGRAM) pr3287 $(DESTDIR)$(BINDIR)/pr3287
-	$(INSTALL_PROGRAM) prtodir $(DESTDIR)$(BINDIR)/prtodir
+	$(INSTALL_SCRIPT) prtodir $(DESTDIR)$(BINDIR)/prtodir
 
 install.man: man
 	[ -d $(DESTDIR)$(MANDIR)/man1 ] || \
